src/objective-c/tests/CronetUnitTests/CronetUnitTests.m

```diff
@@ -270,8 +270,11 @@ - (void)testInternalError {
 }
 
 - (void)PacketCoalescing:(bool)use_coalescing {
-  grpc_cronet_use_packet_coalescing(use_coalescing);
-
+  grpc_arg arg;
+  arg.key = GRPC_ARG_USE_CRONET_PACKET_COALESCING;
+  arg.type = GRPC_ARG_INTEGER;
+  arg.value.integer = use_coalescing ? 1:0;
+  grpc_channel_args *args = grpc_channel_args_copy_and_add(NULL, &arg, 1);
```

Comment:
I'm pretty sure we have somewhere a constructor for `grpc_channel_args` from a `NSDictionary` that you can use to make this clear.